src/Task.java

```diff
@@ -1,19 +1,40 @@
 import java.util.Date;
 
+/**
+ * This class is part of the "Pursuit" application.
+ * "Pursuit" is a very simple, text based To Do List.
+ *
+ * This Task class creating a new task each time and using parameters.
+ *
+ * @author  Tarek Barodi
+ * @version 2019.02.29
+ */
+
 public class Task
 {
     private String title;
     private String description;
     private String project;
     private Date date;
+    private String dueDate;
     private Boolean status;
 
 
-    public Task(String title, String description, String project, Date date, Boolean status) {
```

Comment:
Check the type of the dueDate and follow up, what can we do to activate the due date to be entered correct and send reminder on the Interface page